package.json

```diff
@@ -153,10 +154,11 @@
     "compile-validators": "node ./build/compile-validators.js",
     "lint": "eslint . --ext .ts --max-warnings 0",
     "lint:fix": "eslint . --ext .ts --fix",
-    "test:node": "npm run compile-validators && tsc && c8 mocha \"dist/esm/tests/**/*.spec.js\"",
+    "madge": "madge --circular ./src/index.ts",
+    "test:node": "npm run madge && npm run compile-validators && tsc && c8 mocha \"dist/esm/tests/**/*.spec.js\"",
```

Comment:
I have a slight preference for keeping `madge` out of `test:node` since I use `test:node` often locally during development. I see `madge` similar to `lint` where it is good to have in CI, and would be tedious if it blocked testing locally while I'm still working/debugging locally.

Reply:
The change is easy either way so I'll make that change. Though, I am slightly more on the fence on this one comparing to `lint`, mainly because the changes are more involved if you only found out after you submit a PR that you've introduced a circular dependency, it can be more annoying, but the fault is on the author so fair enough I guess!